---
abstract: Raman spectroscopy is an emerging biophotonic tool for the identification of disease. By probing the unique molecular vibrations that depend on the composition and structure of samples, it provides a wealth of information on a cellular and molecular level of both solid and liquid specimens without the use of external agents such as dyes, stains or radioactive labels. Glioblastomas can be stratified by the presence or absence of mutations in isocitrate dehydrogenase (IDH) 1 or 2. Determination of IDH status is critical for histological diagnosis and clinical decision-making with evidence that the presence of IDH mutation confers better prognosis and a better response to chemotherapy. Mutation-positive tumours accumulate high concentrations of D-2-hydroxyglutarate, resulting in metabolic and epigenetic changes which we hypothesize will be reflected in a change in Raman scattering. In this study, we investigate the feasibility of using Raman spectroscopy to differentiate between IDH1 positive (IDH1 + ) and negative (IDH1 - ) tumours through classification modelling. 
authors:
- Emily Feneberg, Philip D. Charles, Mattéa J. Finelli, <b>Connor Scott</b>, Roman Fischer, Olaf Ansorge, Benedikt M. Kessler, Elizabeth Gray, Kevin Talbot, Martin R. Turner 
date: "2019-12-04"
doi: ""
featured: false
image:
  caption: 'Image credit: Own work'
  focal_point: ""
  preview_only: false
projects: []
publication: ''
publication_short: ""
publication_types:
- "2"
publishDate: "2019-15-05" 
summary:  <b><i> Published Article (Journal of Amyotrophic Lateral Sclerosis and Frontotemporal Degeneration, volume 20, S1, November 2019)</i></b> and <i>Poster -  30th International Symposium on ALS/MND, December 4th - 6th, 2019 </i>
tags:
- Source Themes
title: Characterising the Proteome of the Giant Pyramidal Betz Cell. 
url_code: ""
url_dataset: ""
url_pdf: 
url_poster: ""
url_project: ""
url_slides: ""
url_source: ""
url_video: ""
---
<b>Background:</b>

Despite being, the largest cell in the human brain; the principle neuron in the Primary Motor Cortex and Corticospinal Tract; and one of the first cells susceptible to Motor Neuron Disease (MND); little is known about the biochemical composition of the Betz Cell.
<br>
Due to the lack of biomarkers it is difficult to locate. Its identification can be achieved with non-specific histological stains (A,B,D) or Neurofilament Immuno-stain (C,E) with confirmation from a trained Neuropathologist.
<br>
Discovering biomarkers will help distinguish the Betz cell from it’s microenvironment. Defining it’s proteome will greatly aid our understanding of the Motor Cortex, help us explain why these cells are vulnerable to neurodegeneration in MND. 
<br>
<b>Methods:</b>
<br>
We employed Laser Capture Microdissection coupled with LC-MS/MS - using our novel micro-proteomic approach. A simplified workflow of this process can be seen here: 
<br>

<b>Results:</b>

Using our microproteomic approach we were able to successfully identify ~4,000 proteins from as few as 150 Betz Cell. Of this, 1,700 were able to be quantified and compared to a reference proteome – the Purkinje Cell of the Cerebellum. Of the proteins that were found to be significantly higher in the Betz than in the Purkinje we observed >5 fold increase in GO terms relating to glycolysis, ferric iron transport, NADH dehydrogenase & ATPase activity, with most of the highly expressed proteins found within the mitochondria. Confirming previous assumptions that the Betz cell is highly metabolically active. 

Gene Ontology revealed the proportion of proteins from different classes and a generic overview of their biological function.
Proteins were then analysed to determine how their expression differed against Purkinje cells. From this data we found hundreds of proteins that were significantly differentially expressed in the two cell types.  Of note, we found several known Purkinje Cell biomarkers
From our dataset several proteins were selected to test if
they could be used as potential Betz cell biomarkers. 

The results can be found to the right. All of the selected proteins had higher expression in the Betz cell than the surrounding micro-environment. DST and ATPB out-performed against the others. DST was particular concentrated in Betz cell and allowed clear segmentation from surrounding cells and neuropil.

<b>Discussion and conclusions:</b>
For the first time we have revealed the proteome of the Betz cell - ~4,000 proteins. Our data  confirm previous assumptions in the literature about the function & nature of the Betz cell. Our findings were validated with back-mapping immunohistochemistry to human tissue; from this we successfully identified several new markers to help in Betz Cell identification. Next, we wish to explore how the proteome of these Betz cells are affected in Motor Neuron Disease. 
