---
abstract:
authors:
- <b>Connor Scott</b>, Simon Davis, Benedikt M. Kessler, Roman Fischer, Olaf Ansorge
date: "2019-05-15"
doi: ""
featured: false
image:
  caption: ''
  focal_point: ""
  preview_only: false
projects: []
publication: ''
publication_short: ""
publication_types:
- "2"
publishDate: "2019-05-15" 
summary:  Highlights some of the initial novel proteomic work performed on Betz cell. <b><i> Poster - University of Oxford’s Medical Science Division D.Phil Day 2019</i></b>
tags:
- Source Themes
title: Characterising the Proteome of the Giant Pyramidal Betz Cell. 
url_code: ""
url_dataset: ""
url_pdf: 
url_poster: ""
url_project: ""
url_slides: ""
url_source: ""
url_video: ""
---
<b>Background:</b>

Despite being, the largest cell in the human brain; the principle neuron in the Primary Motor Cortex and Corticospinal Tract; and one of the first cells susceptible to Motor Neuron Disease (MND); little is known about the biochemical composition of the Betz Cell.
Due to the lack of biomarkers it is difficult to locate. Its identification can be achieved with non-specific histological stains (A,B,D) or Neurofilament Immuno-stain (C,E) with confirmation from a trained Neuropathologist.
Discovering biomarkers will help distinguish the Betz cell from it’s microenvironment. Defining it’s proteome will greatly aid our understanding of the Motor Cortex, help us explain why these cells are vulnerable to neurodegeneration in MND. 

<b>Methods:</b><br>
<br>
We employed Laser Capture Microdissection coupled with LC-MS/MS - using our novel micro-proteomic approach. A simplified workflow of this process can be seen here: 
<br>

<b>Results:</b>

Using our microproteomic approach we were able to successfully identify ~4,000 proteins from as few as 150 Betz Cell. Of this, 1,700 were able to be quantified and compared to a reference proteome – the Purkinje Cell of the Cerebellum. Of the proteins that were found to be significantly higher in the Betz than in the Purkinje we observed >5 fold increase in GO terms relating to glycolysis, ferric iron transport, NADH dehydrogenase & ATPase activity, with most of the highly expressed proteins found within the mitochondria. Confirming previous assumptions that the Betz cell is highly metabolically active. 

Gene Ontology revealed the proportion of proteins from different classes and a generic overview of their biological function.
Proteins were then analysed to determine how their expression differed against Purkinje cells. From this data we found hundreds of proteins that were significantly differentially expressed in the two cell types.  Of note, we found several known Purkinje Cell biomarkers
From our dataset several proteins were selected to test if
they could be used as potential Betz cell biomarkers. 

The results can be found to the right. All of the selected proteins had higher expression in the Betz cell than the surrounding micro-environment. DST and ATPB out-performed against the others. DST was particular concentrated in Betz cell and allowed clear segmentation from surrounding cells and neuropil.

<b>Discussion and conclusions:</b><br>

For the first time we have revealed the proteome of the Betz cell - ~4,000 proteins. Our data  confirm previous assumptions in the literature about the function & nature of the Betz cell. Our findings were validated with back-mapping immunohistochemistry to human tissue; from this we successfully identified several new markers to help in Betz Cell identification. Next, we wish to explore how the proteome of these Betz cells are affected in Motor Neuron Disease. 
